dpu-wrappers/dpu-upmem-dpurte-clang.in

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 @OS_DEPRECATION_WARNING@
 DPU_CLANG_DIR="$(dirname "$0")"
-"${DPU_CLANG_DIR}/clang" --target=dpu-upmem-dpurte "$@"
+"${DPU_CLANG_DIR}/clang" --target=dpu-upmem-dpurte -g "$@"
```

Comment:
Why do we need hardcoded debug symbols?

Reply:
In the documentation: https://sdk.upmem.com/2023.2.0/080_AboutDPULLDB.html
> "To be able to debug with as much information as possible, debug symbols are enabled by default in dpu-upmem-dpurte-clang. Make sure not to disabled them when debugging (using -g0 by example)."

It was kind of added artificially via the `addDPUTargetOptions`, forcing to be `limited`.
By this, we could have other `-debug-info-kind` the right way via the command line.